diff --git a/node_modules/react-native-easy-toast/index.js b/node_modules/react-native-easy-toast/index.js
index de95087..00770ec 100755
--- a/node_modules/react-native-easy-toast/index.js
+++ b/node_modules/react-native-easy-toast/index.js
@@ -17,7 +17,6 @@ import {
 } from 'react-native'
 
 import PropTypes from 'prop-types';
-const ViewPropTypes = View.propTypes;
 export const DURATION = {
     LENGTH_SHORT: 500,
     FOREVER: 0,
@@ -146,7 +145,7 @@ const styles = StyleSheet.create({
 });
 
 Toast.propTypes = {
-    style: ViewPropTypes.style,
+    style: PropTypes.style,
     position: PropTypes.oneOf([
         'top',
         'center',
